Modules/Core/Common/include/itkImageToImageFilter.hxx

```diff
@@ -208,7 +208,7 @@ ImageToImageFilter<TInputImage, TOutputImage>::VerifyInputInformation() const
         {
           directionString.setf(std::ios::scientific);
           directionString.precision(7);
-          directionString << "InputImage Direction: " << inputPtr1->GetDirection() << ", InputImage" << it.GetName()
+          directionString << "InputImage Direction: " << inputPtr1->GetDirection() << ", InputImage " << it.GetName()
```

Comment:
For the sake of completeness, you could make similar changes to `ImageSink<TInputImage>::VerifyInputInformation()`, if you want to 🤷